edx-platform/pearson-vue-theme/lms/templates/header/user_dropdown.html

```diff
@@ -3,36 +3,35 @@
 <%namespace name='static' file='static_content.html'/>
 
 <%!
-import json
-
 from django.conf import settings
 from django.urls import reverse
-from django.utils.translation import gettext as _
+from django.utils.translation import ugettext as _
 
 from openedx.core.djangoapps.user_api.accounts.image_helpers import get_profile_image_urls_for_user
 from openedx.core.djangoapps.user_api.accounts.toggles import should_redirect_to_order_history_microfrontend
 from openedx.core.djangoapps.user_api.accounts.utils import retrieve_last_sitewide_block_completed
-from openedx.features.enterprise_support.utils import get_enterprise_learner_generic_name, get_enterprise_learner_portal
+from openedx.features.enterprise_support.utils import get_enterprise_learner_generic_name
+from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
 %>
 
 <%
 ## This template should not use the target student's details when masquerading, see TNL-4895
 self.real_user = getattr(user, 'real_user', user)
 profile_image_url = get_profile_image_urls_for_user(self.real_user)['medium']
 username = self.real_user.username
+
+if configuration_helpers.get_value('SHOW_PROFILE_NAME_INSTEAD_OF_USERNAME', False):
```

Comment:
Are we using this setting in any of the prod sites? @sergivalero20

Reply:
@Squirrel18 yes, in pathways:

![image](https://github.com/Pearson-Advance/openedx-themes/assets/73655023/2de53c4b-ceff-4ebe-bdd1-a624da7b876b)

Reply:
We will no longer support courses-pathways.pearson.com.